eng/pipelines/common/global-build-job.yml

```diff
@@ -65,7 +69,7 @@ jobs:
     - ${{ if eq(parameters.isOfficialBuild, true) }}:
       - template: /eng/pipelines/common/restore-internal-tools.yml
 
-    - ${{ if in(parameters.osGroup, 'OSX', 'iOS', 'tvOS') }}:
+    - ${{ if or(in(parameters.osGroup, 'OSX', 'iOS', 'tvOS'), eq(parameters.osGroup, 'Android')) }}:
```

Comment:
instead just add `Android` to the `in` list, to make the condition simpler.

Reply:
I also added 'Android` to line 85 with the `Disk Usage after Build` build step